block/manager.go

```diff
@@ -397,12 +397,6 @@ func (m *Manager) trySyncNextBlock(ctx context.Context, daHeight uint64) error {
 			return fmt.Errorf("failed to save block responses: %w", err)
 		}
 
-		// SaveValidators commits the DB tx
-		err = m.saveValidatorsToStore(bHeight)
```

Comment:
if you are not planning on using Save and Load validators, you can remove the related code in store for getValidatorsKey and validatorsPrefix in https://github.com/rollkit/rollkit/blob/a10a00a8c19dd326509ac3da02f80a10879bb263/store/store.go#L28

Reply:
Follow up issue: https://github.com/rollkit/rollkit/issues/1344